README.md

```diff
@@ -57,9 +57,9 @@ For example, `InsufficientBalance`.
 
 #### 3. Event names should be past tense.
 
-For example, `UpdatedOwner` not `UpdateOwner`.
+For example, `OwnerUpdated` not `UpdateOwner`.
 
-Events should track things that _happened_ and so should be past tense. Using past tense also helps avoid naming collisions with structs or functions.
+Events should track things that _happened_ and so should be past tense. Using past tense also helps avoid naming collisions with structs or functions. We also recommend following the `SubjectVerb`(e.g. `OwnerUpdated`) format rather than `VerbSubject` (e.g. `UpdatedOwner`).
```

Comment:
Sorry @xenoliss one more comment here. I think `We also recommend...` could be its own rule? Not sure we need to bundle with past tense